js/interface/add-markers.js

```diff
@@ -0,0 +1,393 @@
+import Multiselect from 'vue-multiselect'
+
+Fliplet.Floorplan.component('add-markers', {
```

Comment:
What is the idea behind `Fliplet.Floorplan.component` and not just using Vue.component directly?

Reply:
@angelnikolov 
To be honest I have followed the base of the Form Builder. Nick built the Form Builder so I guess he must have an answer.

However from what I see in the Form Builder and in the code I wrote, we need to do this way because we need to compile our templates before passing it to `Vue.component`. Also allows us to do some checks before doing it. See file `core.js`.

cc @squallstar